### PR TITLE
Pioneer 15a Glider Deployments

### PR DESCRIPTION
**Pull request by cdobs**
**Quick-look report:** 3204-01505

**FCPs:** 
340: 3304-00150-00127
380: 3304-00150-00128
387: 3304-00150-00129

**Deployment Docs:**
387: deployment doc: 3304-00021-00107.docx
380: deployment doc: 3304-00021-00108.docx
340: deployment doc: 3304-00021-00109.docx

I also updated the names of previous deployment and recovery cruise IDs to remove the names that are to be deleted from cruiseInformation.cvs

### CODE REVIEW

File: deployment/CP05MOAS-GL340_Deploy.csv [hunk 1]
```diff
@@ -65,3 +65,9 @@ TS20200925,,EN657,,CP05MOAS-GL340-03-CTDGVM000,11,1,2020-09-26T06:14:00,2020-10-
...
 TS20200925,,EN657,,CP05MOAS-GL340-01-ADCPAM000,11,1,2020-09-26T06:14:00,2020-10-16T21:37:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-72777,40.0667,-70.18002,,200,200,EB
+TS20201121,,,,CP05MOAS-GL340-00-ENG000000,12,1,2020-11-21T19:30:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGCON-GLDRCP-00340,40.08387,-70.36168,,200,200,EB
+TS20201121,,,,CP05MOAS-GL340-02-FLORTM000,12,1,2020-11-21T19:30:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-FLORTM-04467,40.08387,-70.36168,,200,200,EB
+TS20201121,,,,CP05MOAS-GL340-03-CTDGVM000,12,1,2020-11-21T19:30:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-CTDGVM-09363,40.08387,-70.36168,,200,200,EB
+TS20201121,,,,CP05MOAS-GL340-04-DOSTAM000,12,1,2020-11-21T19:30:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-DOSTAM-00361,40.08387,-70.36168,,200,200,EB
+TS20201121,,,,CP05MOAS-GL340-05-PARADM000,12,1,2020-11-21T19:30:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-PARADM-50238,40.08387,-70.36168,,200,200,EB
+TS20201121,,,,CP05MOAS-GL340-01-ADCPAM000,12,1,2020-11-21T19:30:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-72777,40.08387,-70.36168,,200,200,EB
```
Comment (snwhite) [inline deployment/CP05MOAS-GL340_Deploy.csv:73]: Looks good.

File: deployment/CP05MOAS-GL380_Deploy.csv [hunk 2]
```diff
@@ -52,4 +52,10 @@ TS20200613,,TS20200629,,CP05MOAS-GL380-01-ADCPAM000,9,1,2020-06-13T21:24:23,2020
...
\ No newline at end of file
+TS20200613,,TS20200629,,CP05MOAS-GL380-05-PARADM000,9,1,2020-06-13T21:24:23,2020-06-30T00:07:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50151,39.92853,-70.37213,,1000,1000,SS-1
+TS20201121,,,,CP05MOAS-GL380-00-ENG000000,10,1,2020-11-21T17:44:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGCON-GLDRCP-00380,39.888,-70.35503,,1000,1000,SS-1
+TS20201121,,,,CP05MOAS-GL380-01-ADCPAM000,10,1,2020-11-21T17:44:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-ADCPAM-49016,39.888,-70.35503,,1000,1000,SS-1
+TS20201121,,,,CP05MOAS-GL380-02-FLORTM000,10,1,2020-11-21T17:44:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-FLORTM-04465,39.888,-70.35503,,1000,1000,SS-1
+TS20201121,,,,CP05MOAS-GL380-03-CTDGVM000,10,1,2020-11-21T17:44:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-CTDGVM-09027,39.888,-70.35503,,1000,1000,SS-1
+TS20201121,,,,CP05MOAS-GL380-04-DOSTAM000,10,1,2020-11-21T17:44:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-DOSTAM-00195,39.888,-70.35503,,1000,1000,SS-1
+TS20201121,,,,CP05MOAS-GL380-05-PARADM000,10,1,2020-11-21T17:44:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-PARADM-50146,39.888,-70.35503,,1000,1000,SS-1
```
Comment (snwhite) [inline deployment/CP05MOAS-GL380_Deploy.csv:61]: Looks good.

File: deployment/CP05MOAS-GL387_Deploy.csv [hunk 3]
```diff
@@ -29,15 +29,21 @@ SI20170418,,TS20170702,,CP05MOAS-GL387-02-FLORTM000,5,1,2017-04-19T20:53:02,2017
...
+AR34,,WJ20190617,,CP05MOAS-GL387-05-PARADM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50146,39.9681,-71.00002,,1000,1000,FZ-1
+TS20201121,,,,CP05MOAS-GL387-00-ENG000000,8,1,2020-11-21T17:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGCON-GLDRCP-00387,39.88723,-70.35692,,1000,1000,FZ-1
+TS20201121,,,,CP05MOAS-GL387-01-ADCPAM000,8,1,2020-11-21T17:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-ADCPAM-55979,39.88723,-70.35692,,1000,1000,FZ-1
+TS20201121,,,,CP05MOAS-GL387-02-FLORTM000,8,1,2020-11-21T17:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-FLORTM-03884,39.88723,-70.35692,,1000,1000,FZ-1
+TS20201121,,,,CP05MOAS-GL387-03-CTDGVM000,8,1,2020-11-21T17:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-CTDGVM-09055,39.88723,-70.35692,,1000,1000,FZ-1
+TS20201121,,,,CP05MOAS-GL387-04-DOSTAM000,8,1,2020-11-21T17:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-DOSTAM-00145,39.88723,-70.35692,,1000,1000,FZ-1
+TS20201121,,,,CP05MOAS-GL387-05-PARADM000,8,1,2020-11-21T17:40:00,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-PARADM-50324,39.88723,-70.35692,,1000,1000,FZ-1
```
Comment (snwhite) [inline deployment/CP05MOAS-GL387_Deploy.csv:49]: Looks good.

**Review by snwhite — APPROVED** [review]

File: deployment/CP05MOAS-GL340_Deploy.csv (hunk 1 above)
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL340_Deploy.csv:73]: new deployment looks good.  All the CUID updates look good.

File: deployment/CP05MOAS-GL380_Deploy.csv (hunk 2 above)
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL380_Deploy.csv:61]: New deployment looks good.  CUID updates look good.

File: deployment/CP05MOAS-GL387_Deploy.csv (hunk 3 above)
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL387_Deploy.csv:49]: New deployment looks good.  CUID updates look good.  For glider 387 the PAR sensor SN needs to be corrected on FCP from 50234 to 50324 (confirmed that 50324 is correct via vendor cal doc and autoexec file in FCP zip folder).

**Review by rtravis-whoi — APPROVED** [review]

**Pull request merged**